VAMobile/src/screens/HomeScreen/ProfileScreen/PersonalInformationScreen/PersonalInformationScreen.test.tsx

```diff
@@ -43,29 +42,45 @@ jest.mock('../../../../api/demographics/getDemographics', () => {
   }
 })
 
-const authorizedMilitaryState = {
-  militaryService: {
-    ...initialMilitaryServiceState,
-    mostRecentBranch: BranchesOfServiceConstants.AirForce,
-    serviceHistory: [{} as ServiceData],
-  },
-}
-
-const personalInformationState = {
-  ...initialPersonalInformationState,
-  needsDataLoad: false,
-}
+jest.mock('../../../../api/personalInformation/getPersonalInformation', () => {
+  let original = jest.requireActual('../../../../api/personalInformation/getPersonalInformation')
+  return {
+    ...original,
+    usePersonalInformation: jest.fn().mockReturnValue({
+      status: "success",
+      data: {
+        firstName: 'Ben',
+        middleName: 'J',
+        lastName: 'Morgan',
+        signinEmail: '',
+        signinService: '',
+        birthDate: '1990-05-08',
+      }
+    }).mockReturnValueOnce({
+      isLoading: true,
+    }).mockReturnValueOnce({
+      status: "success",
+      data: {
+        firstName: 'Ben',
+        middleName: 'J',
+        lastName: 'Morgan',
+        signinEmail: '',
+        signinService: '',
+        birthDate: '',
+      }
+    })
+  }
+})
```

Comment:
With the contact info PR merged, you can now initialize the data in the query cache via the [render function](https://github.com/department-of-veterans-affairs/va-mobile-app/blob/6521804f5f754beb015253f92c1817587015f82f/VAMobile/src/testUtils.tsx#L246). Alternatively, you could also mock the API client to return this, but since the data will be prefetched by the time the user gets to this screen, initializing it in the query cache probably provides the more accurate experience

Reply:
updated 6112 ticket with these notes. There is a lot of updating I have to do for this test file to make sure everything is working as it should so I feel this is fine for now and I will update it to the render function when I make the updates on that branch